### PR TITLE
[skip ci] Nightly global test status bug fix

### LINKED ISSUE #3161: Nightly global test status is reporting incorrect status
The nightly status is always getting reported as Passed even if a test fails. This is caused due to a bug in how we iterate the nightly status loop to fetch the global status.

### PR DESCRIPTION
**Pull request by rajanashok**
Fixing the bug that reports an incorrect nightly global status.
Fixes #3161

### CODE REVIEW

File: tests/nightly/nightly-kickoff.sh [hunk 1]
```diff
@@ -75,10 +75,13 @@ for i in $nightly_list_var; do
     echo $count
 done
 
-for i in $nightlystatus; do
+for i in "${nightlystatus[@]}" 
+do
+    echo $i
     if [ $i = "Passed" ]
```
Comment (mhagen-vmware) [inline tests/nightly/nightly-kickoff.sh]: I don't think this is still right, you need to check here if the global is ever false and do not set it to pass if it ever becomes false.

Comment (rajanashok) [inline tests/nightly/nightly-kickoff.sh]: I've verified that this works with this following test code.

#!/bin/bash
nightlystatus=( Passed Failed Passed )
count=0

for i in "${nightlystatus[@]}" 
do
    echo $i
    if [ $i = "Passed" ]
    then
    buildStatus=0
    echo "Test Passed!"
    else
    buildStatus=1
    echo "Test failed, setting global test status to Failed!"
    break
    fi	
done

Comment (mhagen-vmware) [inline tests/nightly/nightly-kickoff.sh]: oh, sorry I missed the break statement

**Comment by mhagen-vmware** [comment]
lgtm

**Comment by anchal-agrawal** [comment]
lgtm

**Comment by rajanashok** [comment]
.

**Pull request merged**